source/rails-girls/_coaches.haml

```diff
@@ -0,0 +1,12 @@
+%section
```

Comment:
you have good eyes!!!! Or more probable that I am going color blind - I can't tell the difference